src/lib/preload-libc/syscall_wrappers.c

```diff
@@ -46,6 +46,9 @@ INTERPOSE(bind);
 INTERPOSE(bpf);
 #endif
 // Skipping SYS_brk
+#ifdef SYS_cachestat // kernel entry: num=451 func=sys_cachestat
+INTERPOSE(cachestat);
+#endif
```

Comment:
The wrapper generation script grabs a current list of syscalls and generates wrappers for any that we haven't specifically blocklisted.

Maybe we ought to pin the linux version that it gets this list from so that we don't get random additions like this, or even switch to an allowlist, but leaving it for now.